### PR TITLE
Add recipe for chocpretzels and choco_coffee_beans

### PR DESCRIPTION
**Pull request by Cyrano7**
Adds recipe for chocpretzels and choco_coffee_beans. Pretty easy stuff to do irl.

### CODE REVIEW

**Comment by Coolthulhu** [comment]
The proportions are off:
It produces 9 chocolate pretzels from 3 regular ones.
That's because "result_mult" multiplies the portion size, which is already 3 and 5 for pretzels and coffee beans respectively.

The recipe can't use roasted coffee beans. Is this intended?
5 portions of coffee beans is a lot. You can make 100 units of coffee powder from it. One portion would be fine.

Not sure how large is the chocolate bar supposed to be, but entire bar for 3 pretzels would be an overkill. One bar for 9 pretzels would be more or less OK.

**Comment by Rivet-the-Zombie** [comment]
If we're talking chocolate covered coffee beans, then it should specifically require roasted ones; raw coffee beans aren't fit to eat.

File: data/json/recipes/recipe_food.json [hunk 1]
```diff
@@ -5652,6 +5652,64 @@
...
+    ],
+    [
+      [ "chocolate", 1 ]
+    ]
+  ]
+},{
+  "type" : "recipe",
+  "result": "chocpretzels",
+  "category": "CC_FOOD",
+  "subcategory": "CSC_FOOD_OTHER",
```
Comment (illi-kun) [inline data/json/recipes/recipe_food.json]: I guess it fits to ```CSC_FOOD_SNACK``` well.

**Comment by illi-kun** [comment]
> The recipe can't use roasted coffee beans. Is this intended?

Well, the receipt exactly uses roasted beans but I think I know the reason of this misunderstanding.

**Comment by Cyrano7** [comment]
Okay sry about that. My original intention was about what you guys described. Was a bit late...... Except for CSC_FOOD_SNACK thanks @illi-kun. As for the portions @Coolthulhu So its not clear how big the chocolate bar is but its gives you +10 happiness. 3 pretzels gives you +4 each if I remember correctly. Chocolate pretzels is +9 each. Chocolate coffee beans is +2 each plus caffeine buff.
So the current happiness math is ingredients +22 total and crafted +27 for the chocolate pretzels Ingredients +10 and finished +10 with caffeine buffs for chocolate coffee beans.

I was kinda on the fence on whether its should be doubled though.
As in needing and getting 6 pretzels and getting out 10 beans instead of 5.

9 pretzels per batch seems like a lot though.

**Comment by Coolthulhu** [comment]
> 9 pretzels per batch seems like a lot though. 

The biggest problem would be getting the pretzels, not the balance issues.
Comestibles made out of rare ingredients can have some extra "power" compared to ones made of common stuff.
The problem with buffing batch size is that there is currently no way to make it consume any less and pretzels are neither common nor craftable.

I think the best way to solve it would be to give chocolate 3 or 4 charges (and nerf effects of a single charge) and update recipes to use more sensible amounts of chocolate.
Then pretzels could be crafted in small batch sizes, like 3 per every half or one-third of a chocolate bar.

**Comment by Cyrano7** [comment]
"Comestibles made out of rare ingredients can have some extra "power" compared to ones made of common stuff."

That's a good argument.
double would be +20 extra happiness for the pretzels 
and just getting double the output for the beans

Pretzels are limited in total since you cant make them. But they aren't so hard to find.

**Comment by Chezzo** [comment]
Pretzels aren't craftable? My Philly roots are crying. 

(Though people in Philly DON'T make the kinds of pretzels you put chocolate on.)

**Pull request merged**